doc/workflows/find-diff-action.md

```diff
@@ -0,0 +1,83 @@
+# find-diff-action
+
+This action lets you find which areas of your repository has changed, so you can customize or skip parts of subsequent workflows.
+
+By default all changes are reported, but the action can be configured to only consider certain folders or files and certain depths.
+
+The action produces both a list of relevant changes and a pattern for these changes.
+
+# Usage
+
+## List all changes
+
+```yaml
+- uses: CodeReaper/find-diff-action@v3
```

Comment:
Seems like we are not using this anymore? Isn't diff inside here now?

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1145/commits/6679331224632b3ef0dbabfab8bee5f2ea9b314c